applications/StructuralMechanicsApplication/custom_elements/small_displacement_mixed_volumetric_strain_element.cpp

```diff
@@ -965,9 +908,29 @@ int  SmallDisplacementMixedVolumetricStrainElement::Check(const ProcessInfo& rCu
 {
     KRATOS_TRY
 
-    int base_element_check = SmallDisplacementMixedVolumetricStrainElement::BaseType::Check(rCurrentProcessInfo);
+    int check = SmallDisplacementMixedVolumetricStrainElement::BaseType::Check(rCurrentProcessInfo);
+
+    // Base check
+    check = StructuralMechanicsElementUtilities::BaseElementCheck(this, rCurrentProcessInfo);
+
+    // Verify that the variables are correctly initialized
+    KRATOS_CHECK_VARIABLE_KEY(THICKNESS)
```

Comment:
I think this is already done in the `BaseElementCheck` you have implemented.

Reply:
Typo, should be VOLUMETRIC_STRAIN